cmd/cloud.go

```diff
@@ -343,20 +348,67 @@ func getCmdCloud(gs *state.GlobalState) *cobra.Command {
 	}
 
 	exampleText := getExampleText(gs, `
-  {{.}} cloud script.js`[1:])
+  # [deprecated] Run a k6 script in the Grafana Cloud k6
+  $ {{.}} cloud script.js
+
+  # [deprecated] Run a k6 archive in the Grafana Cloud k6
+  $ {{.}} cloud archive.tar
+
+  # Authenticate with Grafana Cloud k6
+  $ {{.}} cloud login
+
+  # Run a k6 script in the Grafana Cloud k6
+  $ {{.}} cloud run script.js
+
+  # Run a k6 archive in the Grafana Cloud k6
+  $ {{.}} cloud run archive.tar`[1:])
 
 	cloudCmd := &cobra.Command{
 		Use:   "cloud",
 		Short: "Run a test on the cloud",
-		Long: `Run a test on the cloud.
+		Long: `Run a test archive in the Grafana Cloud k6.
 
-This will execute the test on the k6 cloud service. Use "k6 login cloud" to authenticate.`,
-		Example: exampleText,
-		Args:    exactArgsWithMsg(1, "arg should either be \"-\", if reading script from stdin, or a path to a script file"),
+This will execute the test in the Grafana Cloud k6 service. Be sure to run the "k6 cloud login" command prior to
```

Comment:
How about expanding a bit to give more explanations of what is happening? Feel free to ignore suggestion

```suggestion
This will archive test script(s), including all necessary resources, and execute the test in the Grafana Cloud k6 service. Be sure to run the "k6 cloud login" command prior to
```